extensions/amp-next-page/0.1/next-page-service.js

```diff
@@ -497,19 +515,59 @@ export class NextPageService {
 
   /**
    * Sets the specified document as active, updating the document title and URL.
-   * @param {!DocumentRef} documentRef Reference to the document to set as
-   *     active.
+   *
+   * @param {number} documentIndex Index of the document to be activated
    * @private
    */
-  setActiveDocument_(documentRef) {
-    const {amp} = documentRef;
-    this.win_.document.title = amp.title || '';
-    this.activeDocumentRef_ = documentRef;
-    this.setActiveDocumentInHistory_(documentRef);
+  setActiveDocument_(documentIndex) {
+    this.documentRefs_.forEach((docRef, index) => {
+      const {amp} = docRef;
+      let updatedVisibilityState;
+      // Update the title and history
+      if (index == documentIndex) {
```

Comment:
Looks like you use `===` later in the file. I think that is the new recommendation, so should just be consistent with the changes.